.github/workflows/validation.yml

```diff
@@ -19,27 +24,32 @@ jobs:
         with:
           fetch-depth: 1
 
-      - name: Set up Python 3
-        uses: actions/setup-python@v4
-        with:
-          python-version: "3.x"
-
-      - name: Install dependencies
-        run: pip install -r requirements.txt
+      - name: Fetch list of repositories
+        run: |
+          git config --global --add safe.directory /__w/standard-GEM-validation/standard-GEM-validation
+          python -c 'import runner; runner.matrix()'
 
-      - id: set-matrix
-        run: echo "matrix=$(python -c 'import runner; runner.matrix()')" >> $GITHUB_OUTPUT
-
-      - name: Cache pip directory
-        uses: actions/cache@v3
-        id: cache
+      - name: Commit index of standard-GEMs
+        uses: stefanzweifel/git-auto-commit-action@v4
         with:
-          path: ~/.cache/pip
-          key: ${{ github.sha }}
+          commit_user_name: validation-bot
+          commit_message: update index of standard-GEMs
+          file_pattern: index.json
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: Define job matrix from index
+        id: set-matrix
+        run: echo "matrix=$(cat index.json)" >> $GITHUB_OUTPUT
 
   validate:
     needs: setup
     runs-on: ubuntu-latest
+    container:
+      image: ghcr.io/metabolicatlas/memote-docker:0.13
+      volumes:
+        - ${{ github.workspace }}:/project:rw
+      options: --user root --workdir /project
```

Comment:
Where would the output json for each individual model be saved and re-accessed to enable access to the MEMOTE results?

Reply:
From the JSON, I chose to keep only the total resulting score, not the full JSON, which I thought would be too large.